lib/raven/interfaces/exception.rb

```diff
@@ -4,15 +4,12 @@ module Raven
   class ExceptionInterface < Interface
 
     name 'exception'
-    attr_accessor :type
-    attr_accessor :value
-    attr_accessor :module
-    attr_accessor :stacktrace
+    attr_accessor :values
```

Comment:
Now I can't find the old documentation for how Exception interfaces are supposed to work with Sentry. I thought that info was here: http://sentry.readthedocs.org/en/latest/developer/interfaces/ but it isn't anymore. @dcramer where'd that go?